README.md

```diff
@@ -147,3 +148,17 @@ To create a new (empty) package (e.g. spinning Zoe out from ERTP):
 * commit everything to a new branch, push, check the GitHub `Actions` tab to
   make sure CI tested everything properly
 * merge with a PR
+
+# Related repositories
+
+XS JavaScript engine and Endo helps create sandbox environment for Vats.
+
+  * https://github.com/agoric-labs/moddable.git
+  * https://github.com/agoric-labs/xsnap-pub
+  * https://github.com/endojs/endo
+
+# Agoric tutorials
+
+* [Chainboard Academy Bootcamp Lectures](https://www.youtube.com/watch?v=wWQxJnmZttE&list=PLZUbmHar_o80AEV_JjoxLZwxFIc-8uQAq) - [Lecture notes](https://github.com/Chainboard-Academy/agoric-lecture-content)
+* [Agoric Tech Talks](https://www.youtube.com/watch?v=iyuo0ymTt4g&list=PLhuBigpl7lqth_Ow_eQWZs7NFxmeDw9W8)
+* [Agoric Patterns](https://ilyasmercan.github.io/AgoricPatterns/)
```

Comment:
The chainboard academy and Agoric Patterns are 3rd-party materials. While they're very nice, I don't think we should cite them as "Agoric Tutorials". Maybe move them to a [wiki](https://github.com/Agoric/agoric-sdk/wiki) page and/or a [discussion](https://github.com/Agoric/agoric-sdk/discussions)?

Reply:
A renaming of "Agoric Tutorials" to just "Tutorials" ? The third link is also an external partner's link. I thought these and others similar links in future will be good place to add here for anyone to learn more from different angles. Any other suggestion for the heading or should I just remove these external links?

Reply:
"Tutorials" would be enough to address my critical concern. Oh... also, add "by Ilyas Mercan" to the last one to distinguish its authorship from "Agoric Tech Talks".

But by way of suggestion/preference, I don't think they're all tutorials. How about "Learning Resources"?